state-chain/chains/src/eth.rs

```diff
@@ -77,6 +77,43 @@ impl EthereumTrackedData {
 	}
 }
 
+mod fees {
+	// TODO: refine these constants.
```

Comment:
Difficult to set accurate values here, but I'd suggest some changes.
A signature verification in Ethereum will cost ~60-80k normally. And on top of that, the fetch or transfer will be 20-30k on top. It probably makes sense to lower that slightly since there will be some transactions batching multiple actions, so my proposal would be something along the lines of:
```
	pub const BASE_COST_PER_BATCH: u128 = 50_000;
	pub const GAS_COST_PER_FETCH: u128 = 30_000;
	pub const GAS_COST_PER_TRANSFER_NATIVE: u128 = 20_000;
	pub const GAS_COST_PER_TRANSFER_TOKEN: u128 = 40_000;
```
I have added a differentiation between native and token on the egress as explained in the other comment.